docs/learn.md

```diff
@@ -38,9 +38,9 @@ for such things, that works for me, too.)
 
 ## Download It
 
-[Download logstash-%VERSION%](https://download.elasticsearch.org/logstash/logstash/logstash-%VERSION%.tar.gz)
+[Download logstash-%VERSION%](https://download.elastic.co/logstash/logstash/logstash-%VERSION%.tar.gz)
 
 ## What's next?
 
-Try this [guide](tutorials/getting-started-with-logstash) for a simple
-real-world example getting started using logstash.
+Try this [guide](http://www.elastic.co/guide/en/logstash/current/getting-started-with-logstash.html) for a simple
```

Comment:
same comment here; we don't want users reading the 1.4.3 docs to be pointed at the `current` guide docs, since that will confuse them.